integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -125,6 +126,19 @@ def clean_content(self):
             link.attrib.pop("target", None)
             self.logger.debug("Removed target attribute from link: %r", tostring(link))
 
+        # Update internal links
+        for link in content.iter("a"):
+            if href := link.attrib["href"]:
```

Comment:
```suggestion
            if href := link.attrib.get("href"):
```
I think an invalid link object without a `href` attribute should not cause an internal server error.